core/tests/protractor.conf.js

```diff
@@ -291,8 +291,8 @@ exports.config = {
         // Directory for screenshots
         dest: '../protractor-screenshots',
         // Function to build filenames of screenshots
-        filename: function(spec, descriptions, results, capabilities) {
-          return descriptions[1] + ' ' + descriptions[0];
+        pathBuilder: function(currentSpec, suites) {
+          return currentSpec.fullName;
```

Comment:
`filename` is expected to be a string not function, so I found a replacement `pathBuilder` ([ref.](https://www.npmjs.com/package/protractor-jasmine2-screenshot-reporter#path-builder-optional))

These changes will create a filename as describe_message + it_message.png

![image](https://user-images.githubusercontent.com/16653571/75732470-9514fb80-5d18-11ea-92af-96ac899db8b0.png)